pages/accountPage.js

```diff
@@ -14,102 +16,61 @@ const SELECTORS = {
+            });
+            break;
     }
-};
-
-const clickAddNewAddress = async (page) => {
-        const optionElement = await page.waitForXPath(SELECTORS.NAVIGATION_OPTIONS.ADD_NEW_ADRESS, {
-            visible: true,
-            timeout: COMMON_TIMEOUT
-        });
-        await optionElement.click();
-};
-
-const setFirstNameDetail = async (page, firstName) => {
-    const firstNameInput = await page.waitForXPath(SELECTORS.ADDRESS_DETAILS.FIRSTNAME_INPUT);
-    await firstNameInput.click({clickCount: 3})
-    await firstNameInput.type(firstName);
-};
-
-const setLastNameDetail = async (page, lastName) => {
-    const lastNameInput = await page.waitForXPath(SELECTORS.ADDRESS_DETAILS.LASTNAME_INPUT);
-    await lastNameInput.click({clickCount: 3})
-    await lastNameInput.type(lastName);
-};
-
-const setPhoneNumberDetail = async (page, phoneNumber) => {
-    const phoneNumberInput = await page.waitForXPath(SELECTORS.ADDRESS_DETAILS.PHONENUMBER_INPUT);
-    await phoneNumberInput.click();
-    await phoneNumberInput.type(phoneNumber);
-};
 
-const setStreetAddressDetail = async (page, adressDetail) => {
-    const streetAdressInput = await (page.waitForXPath(SELECTORS.ADDRESS_DETAILS.STREETADRESS_INPUT));
-    await streetAdressInput.click();
-    await streetAdressInput.type(adressDetail);
+    await optionElement.click();
+    await page.waitForNavigation({timeout: COMMON_TIMEOUT, waitUntil: 'networkidle2'})
 };
 
-const setCityDetail = async (page, cityDetail) => {
-    const cityInput = await (page.waitForXPath(SELECTORS.ADDRESS_DETAILS.CITY_INPUT));
-    await cityInput.click();
-    await cityInput.type(cityDetail);
-};
-
-const setPostalCode = async(page, postalCode) => {
-    const setPostalCode = await (page.waitForXPath(SELECTORS.ADDRESS_DETAILS.POSTALCODE_INPUT));
-    await setPostalCode.click();
-    await setPostalCode.type(postalCode);
-};
-
-const chooseCountry = async(page, country_Id) => {
-    const countryDropDown = await (page.waitForXPath(SELECTORS.ADDRESS_DETAILS.COUNTRY_DROPDOWN));
-    await countryDropDown.click();
-    await page.waitForTimeout(1000);
-    await countryDropDown.select(country_Id);
-    await page.waitForTimeout(1000);
-};
-
-const chooseProvince = async (page, region_Id) => {
-    const provinceDropDown = await (page.waitForXPath(SELECTORS.ADDRESS_DETAILS.STATEPROVINCE_DROPDOWN));
-    await provinceDropDown.click();
-    await page.waitForTimeout(1000);
-    await provinceDropDown.select(region_Id);
-    await page.waitForTimeout(1000);
-};
-
-const saveAddress = async (page, optionName) => {
-    if (optionName === "Save Address") {
-        const optionElement = await page.waitForXPath(SELECTORS.ADDRESS_DETAILS.SAVEADRESS_BUTTON,  {
-            visible: true,
-            timeout: COMMON_TIMEOUT
+/**
+ * Gets received text from updating Newsletter.
+ * @param page Represents page object of currently handled tab.
+ * @returns {Promise<*>} Returns the received text.
```

Comment:
It is obvious that it returns received text, you have to specify what exactly it returns e.x: Returns newsletter description text.